### PR TITLE
[#222] Set ADMIN_KW when user applying metadata is a rodsadmin (main)

### PR DESCRIPTION
**Pull request by korydraughn**

### CODE REVIEW

**Review by alanking — COMMENTED** [review]
Seems good to me. Should we add a test?

**Comment by trel** [comment]
agreed.    but we should see it behave in a test.   this repo needs to be handsfree/trustworthy once it works.

**Comment by korydraughn** [comment]
I was able to reproduce the `CAT_NO_ACCESS_PERMISSION` error using the test on main. Adding in the fix resolves that issue resulting in a passing test.

**Comment by trel** [comment]
love it.

File: packaging/test_plugin_unified_storage_tiering.py [hunk 1]
```diff
@@ -407,6 +407,57 @@ def test_single_quote_data_name__127(self):
...
+                        # test stage to tier 2.
+                        sleep(15)
+                        invoke_storage_tiering_rule()
+                        delay_assert_icommand(alice_session, f'ils -L {filename}', 'STDOUT_SINGLELINE', 'rnd2')
+
+                        # capture the access time.
+                        _, out, _ = admin_session.assert_icommand(
+                            ['iquest', '%s', f"select META_DATA_ATTR_VALUE where DATA_NAME = '{filename}' and META_DATA_ATTR_NAME = 'irods::access_time'"], 'STDOUT')
+                        access_time = out.strip()
+                        #print(f'access_time = [{access_time}]') # Debug
```
Comment (trel) [inline packaging/test_plugin_unified_storage_tiering.py:441]: remove

File: packaging/test_plugin_unified_storage_tiering.py [hunk 2]
```diff
@@ -407,6 +407,57 @@ def test_single_quote_data_name__127(self):
...
+                        self.assertGreater(len(access_time), 0)
+
+                        # show the access time is updated correctly.
+                        lib.create_ufs_resource(admin_session, resc_name)
+                        admin_session.assert_icommand(f'irepl -M -R {resc_name} {alice_session.home_collection}/{filename}')
+
+                        _, out, _ = admin_session.assert_icommand(
+                            ['iquest', '%s', f"select META_DATA_ATTR_VALUE where DATA_NAME = '{filename}' and META_DATA_ATTR_NAME = 'irods::access_time'"], 'STDOUT')
+                        new_access_time = out.strip()
+                        #print(f'new_access_time = [{new_access_time}]') # Debug
```
Comment (trel) [inline packaging/test_plugin_unified_storage_tiering.py:451]: remove

**Review by trel — APPROVED** [review]
remove those two lines if you want - i see they're for debugging.

otherwise, looks good.

File: packaging/test_plugin_unified_storage_tiering.py (hunk 1 above)
Comment (korydraughn) [reply packaging/test_plugin_unified_storage_tiering.py]: Done

File: packaging/test_plugin_unified_storage_tiering.py (hunk 2 above)
Comment (korydraughn) [reply packaging/test_plugin_unified_storage_tiering.py]: Done

**Review by alanking — APPROVED** [review]
It does indeed test the behavior. Looks good

**Comment by trel** [comment]
that was close.

**Pull request merged**